README.md

```diff
@@ -20,12 +20,28 @@ Currently we support three cloud providers: Exoscale, Safespring, an CityCloud.
 - [jq](https://github.com/stedolan/jq) (tested with jq-1.6)
 - [sops](https://github.com/mozilla/sops) (tested with 3.6.1)
 - [s3cmd](https://s3tools.org/s3cmd) (tested with 2.0.2)
-- [ansible](https://www.ansible.com) (tested with 2.5.1)
+- [ansible](https://www.ansible.com) (tested with 2.9.14)
 - [go](https://golang.org) (tested with 1.13.8)
 
 Note that you will need a [BaseOS](https://github.com/elastisys/ck8s-base-vm) VM template available at your cloud provider of choice!
 See the [releases](https://github.com/elastisys/ck8s-base-vm/releases) for available VM images that can be uploaded to the cloud provider.
 
+#### Installing ansible
+
+You can run the following to install ansible on your machine
+
+```bash
+# On Ubuntu 18.04
+sudo apt update
+sudo apt install software-properties-common
+sudo add-apt-repository --update ppa:ansible/ansible-2.9
+sudo apt install ansible
+
+# On Ubuntu 20.04 - 2.9.6+dfsg-1 is directly available in packages
+sudo apt update
+sudo apt install ansible
+```
+
```

Comment:
I don't really feel this is the place to put package installation instructions. If you don't know how to install the dependencies we do link the various web resources in the requirements list above.

Reply:
Done. Remove this section :)